hexa/workspaces/schema/mutations.py

```diff
@@ -349,14 +349,15 @@ def resolve_create_workspace_connection(_, info, **kwargs):
     mutation_input = kwargs["input"]
 
     try:
-        workspace = Workspace.objects.filter_for_user(request.user).get(
-            slug=mutation_input.pop("workspaceSlug")
-        )
-        mutation_input["connection_type"] = mutation_input.pop("type")
+        with transaction.atomic():
+            workspace = Workspace.objects.filter_for_user(request.user).get(
```

Comment:
This can be done outside of the transaction no ?

Reply:
Yes, it's possible.

Reply:
My point is just that since we do not try to protect `Workspace.objects.filter...` by an atomic transaction, it's better to move it two lines above.

Reply:
Oh yes, my bad. Thx